Bottom: 2e0d2b8430c5389b44e7e760156b801261dd380a
Top:    074b4e3fcf09728ca182e38e33775f9c614acc5b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-23 14:41:38 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 61eacba..8f698f4 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -283,6 +283,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
